### PR TITLE
Update zh-CN translations

### PR DESCRIPTION
**Pull request by auroursa**
Translated the latest messages.po file as usual, and fixed a few inappropriate translation strings, as well as added a few missing periods.

### CODE REVIEW

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:26]: @quiple @tkusano Hi there, since East Asian languages don't have plural forms, I think we can simplify it to this form. What do you think? It works on Simplified Chinese.

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:111]:
```suggestion
msgstr "<0>{0}</0> 关注者"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:115]:
```suggestion
msgstr "<0>{0}</0> 正在关注"
```

File: src/locale/locales/zh-CN/messages.po
Comment (quiple) [reply src/locale/locales/zh-CN/messages.po:26]: Oh, I don't know why, but I noticed that the commas in the numbers disappeared (like `Liked by 7,015 users`) when I typed it this way.

It's probably better to write it like `{0, plural, other {#}}`.

Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po:26]: Thanks for your feedback, this is indeed an issue. Until I find the specific reason, I will use `{0, plural, other {#}}`.

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:22]:
```suggestion
msgstr "其他 {0, plural, {{formattedCount}}} 人"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:26]:
```suggestion
msgstr "{0, plural, other {#}} 个标签已应用到此账户"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:30]:
```suggestion
msgstr "{0, plural, other {#}} 个标签已应用到此内容"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:34]:
```suggestion
msgstr "{0, plural, other {#}} 条转发"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:48]:
```suggestion
msgstr "喜欢（{0, plural, other {#}} 个喜欢）"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:56]:
```suggestion
msgstr "{0, plural, other {#}} 位用户喜欢"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:64]:
```suggestion
msgstr "回复（{0, plural, other {#}} 个回复）"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:72]:
```suggestion
msgstr "取消喜欢（{0, plural, other {#}} 个喜欢）"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:76]:
```suggestion
msgstr "{count, plural, other {#}} 位用户喜欢"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:80]:
```suggestion
msgstr "时"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:84]:
```suggestion
msgstr "分"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:95]:
```suggestion
msgstr "{likeCount, plural, other {#}} 个用户喜欢"
```

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [reply src/locale/locales/zh-CN/messages.po]: @auroursa @quiple 

image


Try this it might be work

**Comment by auroursa** [comment]
@quiple For some reason, I found replacing {0} with {#} resolves the numbers issue. I've only tested it locally and haven't pushed the code yet, but I wanted to share this idea with you.

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [reply src/locale/locales/zh-CN/messages.po]: Update this is wrong

Comment (KuwaLee) [reply src/locale/locales/zh-CN/messages.po]: @quiple can you show me which case or post can find `Liked by 7,015 users` I search all around and no where can i find a case it will show

Comment (KuwaLee) [reply src/locale/locales/zh-CN/messages.po]: @auroursa @quiple I have been viewed carefully and viewed the code and find out the truth

image

The former({0}) is used as a number, while the latter({likeCount}) is a string, apply it by the logic

this version will work!!!!!!!

**Comment by auroursa** [comment]
Even in languages without plurals, omitting the singular form (one) can still lead to unexpected errors, so it's better to keep the original format.

**Comment by auroursa** [comment]
I manually ran intl:extract (only for zh-CN) and added the new translations, so there appear to be a lot of changes. However, I compiled and tested it, and it works fine with the latest code.
image

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [inline src/locale/locales/zh-CN/messages.po:1628]:
```suggestion
msgstr "已到末尾"
```
image

這裡也會顯示所以已到結尾會更好

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [inline src/locale/locales/zh-CN/messages.po:1831]:
```suggestion
msgstr "创建资讯源只须一些编程基础。查看 <0/> 以获取详情。"
```
原文的意思是說"您不需要高深的程式也可以…"

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [inline src/locale/locales/zh-CN/messages.po:3679]:
```suggestion
msgstr "原因：{0}"
```

File: src/locale/locales/zh-CN/messages.po
Comment (KuwaLee) [inline src/locale/locales/zh-CN/messages.po:1143]:
```suggestion
msgstr "无法加载用户个人资料，请稍后再试。"
```
"加載用戶個人資料" 也許比較適合，因為是出現在開始新對話時搜尋profile無法加載

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1831]:
```suggestion
msgstr "创建资讯源仅需你掌握一点编程基础。查看 <0/> 以获取详情。"
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1831]:
```suggestion
msgstr "创建资讯源只需你掌握一点编程基础。查看 <0/> 以获取详情。
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po]: 我重新翻译为“只需掌握一点编程基础”，简体中文里“须”一般解释为“必须”，而“需”是“需要”，所以“只需”会更常用

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:4997]:
```suggestion
msgstr "这里是空的！"
```

**Comment by auroursa** [comment]
I will remove the translation for `{0} your feeds` as it doesn't work well. I'll retranslate it after #3962 is merged.

**Comment by pfrazee** [comment]
Is this one good to go?

**Comment by auroursa** [comment]
@pfrazee yeah, it's ready to go.

**Comment by auroursa** [comment]
Oh hold on, Let me resolve some merge conflicts. When it's ready, I'll let you know. Thanks for your patience.

**Comment by auroursa** [comment]
@pfrazee Yes it's ready to go.

**Comment by auroursa** [comment]
PR #3962 has been merged, so I've retranslated a few entries based on it to ensure the localization is accurate.
I will freeze the translation at this version to prevent merge conflicts while waiting for the merge.

**Review by KuwaLee — COMMENTED** [review]
Fix special case "WhoCanReply"

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:3168]:
```suggestion
msgid "Only {0} can reply."
```

File: src/locale/locales/zh-CN/messages.po
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po]:
```suggestion
msgstr "只有{0}可以回复。"
```

**Review by auroursa — COMMENTED** [review]
fix typo

**Comment by auroursa** [comment]
Thanks @KuwaLee

**Comment by auroursa** [comment]
Please excuse my hasty commit message, it's just some cleanup and minor updates.
I've updated the translation to the latest version and it's ready to be merged.

**Review by pfrazee — APPROVED** [review]
Thanks so much!!

**Pull request merged**
